tests/test_create.py

```diff
@@ -337,6 +337,8 @@ def test_noarch_python_package_with_entry_points(self):
             if on_win:
                 exe_path += ".exe"
             assert isfile(exe_path)
+            output = check_output([exe_path, "--help"], text=True)
+            assert "Usage: flask" in output
```

Comment:
This is the new test that uncovered the Unicode / entry point / launcher hell on Windows.

Reply:
Yikes, that's so simple!